spec/ics-002-client-semantics/README.md

```diff
@@ -253,10 +253,18 @@ but they must expose this common set of query functions to the IBC handler.
 type ClientState = bytes
 ```
 
-Client types must also define a method to initialize a client state with a provided consensus state:
+Client types MUST define a method to initialize a client state with a provided consensus state, writing to state as appropriate.
 
 ```typescript
-type initialize = (state: ConsensusState) => ClientState
+type initialize = (state: ConsensusState) => ()
+```
+
+Client types MUST define a method to fetch the current height (height of the most recent validated header).
+
+```typescript
+type latestHeight = (
```

Comment:
this was confusing for me at least in the beginning. The naming of the third-party client consensus state vs the **self** consensus state was not clear. So whenever it said `getConsensusState(height)` it meant self consensus state (i.e block header and validator info), while the `getConsensusState(clientID, height)` meant the counterparty client.

I'd suggest adding the `client` or `self` prefix to the `ConsensusState` to clarify this item.

Reply:
```suggestion
type latestClientHeight = (
```

Reply:
or, alternatively `clientState.latestHeight()`.

Reply:
Aye, that makes sense, I'll change the client function name to `getClientConsensusState`.

Reply:
Actually, I can't find any uses of `getConsensusState` now which take a `clientID` and `height`. Can you point to a specific location which is confusing?